operatorcourier/validate.py

```diff
@@ -167,20 +172,54 @@ def _csv_spec_validation(self, spec, bundleData):
                     pass
 
             if "owned" not in customresourcedefinitions:
-                self._log_error("spec.customresourcedefinitions.owned"
+                self._log_error("spec.customresourcedefinitions.owned "
                                 "not defined for csv")
-                valid = False
-            else:
-                for crd in customresourcedefinitions["owned"]:
-                    if "name" not in crd:
-                        self._log_error("name not defined for item in "
-                                        "spec.customresourcedefinitions.")
-                        valid = False
-                    else:
-                        if crd["name"] not in crdList:
-                            self._log_error("custom resource definition referenced "
-                                            "in csv not defined in root list of crds")
+                return False
+
+            for csvOwnedCrd in customresourcedefinitions["owned"]:
+                if "name" not in csvOwnedCrd:
+                    self._log_error("name not defined for item in "
+                                    "spec.customresourcedefinitions.")
+                    valid = False
+                elif csvOwnedCrd["name"] not in crdList:
+                    self._log_error("custom resource definition referenced in csv "
```

Comment:
Can this print the name of the CRD that doesn't match?

Reply:
Can the CRD name also be added to the other error logs here as well?

Reply:
Actually I didn't change much logic in those if statements except making the code less indented.

> Can this print the name of the CRD that doesn't match?

Sure I can do that.

> Can the CRD name also be added to the other error logs here as well?

I can but that will require some additional nested `if` checks to make sure `csvOwnedCrd["name"]` exists and is valid. Do you want me to do that?